oletools/olevba.py

```diff
@@ -1888,29 +1890,28 @@ def __init__(self, ole, vba_root, project_path, dir_path, relaxed=True):
                     log.debug("reference type = {0:04X}".format(check))
 
             if check == 0x0033:
```

Comment:
If I can get some examples of how to have other reference types then happy to add them here.